lib/lightning_web/live/credential_live/index.ex

```diff
@@ -43,5 +43,12 @@ defmodule LightningWeb.CredentialLive.Index do
 
   defp list_credentials(user_id) do
     Credentials.list_credentials_for_user(user_id)
+    |> Enum.map(fn c ->
+      project_names =
+        Map.get(c, :projects, [])
+        |> Enum.map_join(", ", fn p -> p.name end)
```

Comment:
While I'm not completely grossed out by that :sweat_smile:, I'm assuming all the projects are preloaded here.

Another way to do this would be via a query 

```
from([c, p] in credential_query,
  select_merge: %{
    project_names: fragment(
      "STRING_AGG(?, ', ') AS project_names",
      p.name
    )
  })
```

Something like that.. I don't think the frequency of this specific page warrants a lot of optimization (beyond ensuring projects are preloaded) for now.